compiler/src/test/kotlin/software/amazon/lastmile/kotlin/inject/anvil/processor/ContributesBindingProcessorTest.kt

```diff
@@ -481,6 +483,41 @@ class ContributesBindingProcessorTest {
         }
     }
 
+    @Test
+    fun `qualifier applied to class annotated with ContributesBinding is carried over to generated code`() {
```

Comment:
Could you please add a test for multibindings too?

Reply:
Added! bd3ba0fec81137a732777b04b9ab02ae73c29aaa

Reply:
I also attempted a fix in 76c7db5b4ab56b6bf100a69536be0a64ff552593 👍